docker-compose.yml

```diff
@@ -4,9 +4,19 @@ services:
     image: postgres:11.7-alpine
     volumes:
       - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
-    ports:
-      - "5432:5432"
     environment:
       POSTGRES_DB: i_have_i_need_development
       POSTGRES_USER: dev_local_user
-      POSTGRES_PASSWORD: password
\ No newline at end of file
+      POSTGRES_PASSWORD: password
+
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    command: bash -c "rm -f tmp/pids/server.pif && bundle exec rails s -p 3000 -b '0.0.0.0'"
```

Comment:
Why the duplication of removing `server.pid`? 
My understanding is that having it in the `entrypoint.sh` was enough to solve the issue.

Reply:
Haha yeah good catch I spelt the filename wrong anyway so that wasn't doing much! I'll take it out